### PR TITLE
MERGE INVITE:

### LINKED ISSUE #32: FIX RECV WRONG RETURN CHECKS:
When a connected client does not send anything, because recv == 0 is a check we do, we do not allow clients
to send 0 bytes.
Therefore shutting down the connection of all trafic incoming from netcat and also from any suspended process.

(This will be dealt with in aw/invite)

### PR DESCRIPTION
**Pull request by AdrianWeaver**
Adding invite feature.

Fix #32 
Still needs to be updated after mode.

Co-authored-by: aweaver [aweaver@student.42.fr](mailto:aweaver@student.42.fr)
Co-authored-by: elouisia [elouisiade@live.fr](mailto:elouisiade@live.fr)

### CODE REVIEW

File: Invite.cpp [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+}
+
+Invite::~Invite()
+{
+#if SHOW_CONSTRUCTOR
+	std::cout << "Invite destructor" << std::endl;
+#endif
+}
+
+Invite::Invite(Server &server, Client &user, std::string rawInput) : ACommand(server, user, rawInput)
```
Comment (Elena-Lou) [inline Invite.cpp:16]: Arg... could you please change user to author please ? xD Sorryyyyyyy

File: Invite.hpp [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# include "irc.hpp"
+# include "ACommand.hpp"
+
+class Client;
+class Server;
+class Invite : public ACommand
+{
+	public:
+		Invite(Server &server, Client &user, std::string rawInput);
```
Comment (Elena-Lou) [inline Invite.hpp:12]: same here

**Review by Elena-Lou — CHANGES_REQUESTED** [review]
Thank you for doing this command \o/

File: Invite.cpp (hunk 1 above)
Comment (AdrianWeaver) [reply Invite.cpp]: fixed!

**Review by Elena-Lou — APPROVED** [review]
Thanks mate !

**Pull request merged**